.github/workflows/docker-build.yml

```diff
@@ -9,27 +9,20 @@ on:
 jobs:
   setup:
     name: Setup
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     outputs:
       modified: ${{ steps.set-modified.outputs.modified }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-
     steps:
       - uses: actions/checkout@v2
       - name: Changed Files Exporter
         id: files
         uses: futuratrepadeira/changed-files@v3.3.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Get a list of modified files
         run: echo "modified=${{ steps.files.outputs.files_created }} ${{ steps.files.outputs.files_updated }}" >> $GITHUB_ENV
-
       - name: Debug Step
```

Comment:
I wonder if we need this "debug step" in the final version ?

Reply:
No, I think no